R/axis.R

```diff
@@ -22,105 +22,103 @@
 #'   geom_point() +
 #'   easy_rotate_x_labels()
 easy_rotate_x_labels <- function(angle = 90, side = c("left", "middle", "right"), ...) {
-    hjust <- switch(
-        match.arg(side),
-        left = 0,
-        middle = 0.5,
-        right = 1
+  hjust <- switch(
+    match.arg(side),
+    left = 0,
+    middle = 0.5,
+    right = 1
+  )
+  angle <- if (is.numeric(angle) | is.integer(angle)) {
+    angle
+  } else if (is.character(angle)) {
+    switch(angle,
+      startattop = -90,
+      startatbottom = 90
     )
-    angle <- if (is.numeric(angle) | is.integer(angle)) {
-       angle
-    } else if (is.character(angle)) {
-       switch(angle,
-              startattop = -90,
-              startatbottom = 90)
-    }
-    ggplot2::theme(axis.text.x = ggplot2::element_text(angle = angle, hjust = hjust))
+  }
+  ggplot2::theme(axis.text.x = ggplot2::element_text(angle = angle, hjust = hjust))
 }
 
-#' Easily remove axe(s)
+#' Easily remove one or more axes
 #'
 #' easy_remove_axes by default removes both axes, but can remove only x or y
 #' if "x" or "y" is given to the 'which' argument
-#' 
+#'
 #' easy_remove_x_axis and easy_remove_y_axis remove just the x or y axis,
 #' respectively.
 #'
 #' @md
 #' @param which which axis or axes to remove, by default "both"
-#' @param what what axis components to remove
-#' @param teach print longer equivalent ggplot expression?
+#' @param what axis components to remove
+#' (`"ticks"`, `"title"`, `"text"`, and/or `"line"`)
+#' @param teach print longer equivalent \code{\link[ggplot2]{ggplot2}} 
+#' expression?
 #'
-#' @return  a \code{\link[ggplot2]{theme}} object  which can be used in 
-#' \code{\link[ggplot2]{ggplot2}} calls 
+#' @return  a \code{\link[ggplot2]{theme}} object  which can be used in
+#' \code{\link[ggplot2]{ggplot2}} calls
 #' @export
 #' @author Alicia Schep
-#' @import rlang
-#' @import ggplot2
 #'
 #' @examples
-#' 
+#'
+#' library(ggplot2)
 #' # Remove all axes
 #' ggplot(mtcars, aes(wt, mpg)) +
 #'   geom_point() + easy_remove_axes()
-#' 
+#'
 #' # remove just x axis
 #' ggplot(mtcars, aes(wt, mpg)) +
 #'   geom_point() + easy_remove_x_axis()
-#'   
+#'
 #' # can also use:
 #' ggplot(mtcars, aes(wt, mpg)) +
 #'   geom_point() + easy_remove_axes("x")
-#'   
+#'
 #' # Remove y axis
 #' ggplot(mtcars, aes(wt, mpg)) +
 #'   geom_point() + easy_remove_y_axis()
-#'   
+#'
 #' # Remove just the ticks
 #' # Remove y axis
 #' ggplot(mtcars, aes(wt, mpg)) +
 #'   geom_point() + easy_remove_y_axis(what = "ticks)
```

Comment:
Missing `"` at the end of `ticks`. Warning on build.

Reply:
thanks for catching this